src/Nancy/Culture/DefaultCultureService.cs

```diff
@@ -31,7 +34,7 @@ public CultureInfo DetermineCurrentCulture(NancyContext context)
 
             foreach (var convention in this.cultureConventions)
             {
-                culture = convention(context);
+                culture = convention(context, this.configuration);

```

Comment:
Only thing I'm a bit unsure of is this. The `ICultureService` interface contract is `CultureInfo DetermineCurrentCulture(NancyContext context)` but I use a `CultureConfiguration` that is injected in and is required to determine current culture however I think it would hard to pass that configuration as a method arg as it is called from `DefaultNancyContextFactory` using `ICultureService`. There's a couple of options but I don't like any of them